src/test/java/org/opensearch/security/UtilTests.java

```diff
@@ -155,6 +155,44 @@ public void testEnvReplace() {
         assertTrue(checked);
     }
 
+    @Test
+    public void testEnvReplacePBKDF2() {
+        Settings settings = Settings.builder().put(ConfigConstants.SECURITY_PASSWORD_HASHING_ALGORITHM, ConfigConstants.PBKDF2).build();
+        final PasswordHasher passwordHasherPBKDF2 = PasswordHasherFactory.createPasswordHasher(settings);
+        assertThat(SecurityUtils.replaceEnvVars("abv${env.MYENV}xyz", settings), is("abv${env.MYENV}xyz"));
+        assertThat(SecurityUtils.replaceEnvVars("abv${envbc.MYENV}xyz", settings), is("abv${envbc.MYENV}xyz"));
+        assertThat(SecurityUtils.replaceEnvVars("abv${env.MYENV:-tTt}xyz", settings), is("abvtTtxyz"));
+        assertTrue(passwordHasherPBKDF2.check("tTt".toCharArray(), SecurityUtils.replaceEnvVars("${envbc.MYENV:-tTt}", settings)));
```

Comment:
Just so I can understand this a little bit more, what is envbc? Do these env variables automatically get hashed?

Reply:
Also do we want to add a test asserting the behavior of the Bcrypt hasher?

Reply:
> Also do we want to add a test asserting the behavior of the Bcrypt hasher?

BCrypt has previously been the only hashing method and is still the default. All the existing tests run through the BCrypt hasher so new ones are not required.

Reply:
@derek-ho Would you be happy for me to mark this conversation as resolved or is there anything else I can provide?

Reply:
@derek-ho Can this thread be marked as resolved?

Reply:
Yes I can't resolve it, please resolve it, thanks!